modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/OrderButton.js

```diff
@@ -41,7 +41,7 @@ function OrderButton() {
 		<div className="mini-cart-submit">
 			<ClayButton
 				block
-				disabled={!cartItems.length}
+				disabled={!cartItems.length || hasPriceOnApplication(cartItems)}
```

Comment:
move disable logic to a new parameter of this component

Reply:
Done